test/rest.test.js

```diff
@@ -2046,6 +2046,33 @@ describe('strong-remoting-rest', function() {
       .end(done);
   });
 
+  it('retain content-type with no-content', function(done) {
+    var method = givenSharedStaticMethod(
+      function(arg, cb) { cb(); },
+      { accepts: { arg: 'arg', type: 'Model' } });
+
+    request(app)
+      .get(method.url + '?arg={"x":1}&_format=xml')
+      .expect(204)
+      .end(function(err, res) {
+        expect(res.get('Content-type')).to.match(/xml/);
+        done();
+      });
+  });
+  it('default content-type to be application/json', function(done) {
```

Comment:
Also change the test name to read like a sentence please. E.g. `it('defaults content-type to application/json')`